cmd/contour/ingressstatus.go

```diff
@@ -74,7 +74,7 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 			}
 			ingressInformers.Wait()
 
-			isw.log.Info("Received a new address for status.loadBalancer")
+			isw.log.WithField("loadbalancer", lbAddress(lbs)).Info("Received a new address for status.loadBalancer")
```

Comment:
How about calling the field name `loadbalancer-address`, then using the same field name in the start up log, so they can be easily correlated?

Reply:
`loadbalancer-address` is a better name, agreed.